test/tvm-ub1804.json

```diff
@@ -7,7 +7,7 @@
       "os_name": "Ubuntu18.04",
       "vm_size": "Standard_D2s_v3",
       "security_profile": {
```

Comment:
if both secureboot and vtpm are true by default, we don't need this section

Reply:
Resolved this in the new iteration.